lib/web_ui/skwasm/text/paragraph.cpp

```diff
@@ -58,6 +61,46 @@ SKWASM_EXPORT int32_t paragraph_getPositionForOffset(Paragraph* paragraph,
   return position.position;
 }
 
+SKWASM_EXPORT bool paragraph_getClosestGlyphInfoAtCoordinate(
+    Paragraph* paragraph,
+    SkScalar offsetX,
+    SkScalar offsetY,
+    SkScalar* graphemeLayoutBounds,
```

Comment:
I think just making this a `SkRect *`, returning a single rect is a little more expressive. `SkRect` is a plain old data struct with four floats, it's pretty safe to assume the memory layout. (We already do this for `SkRect` objects in several places).